### PR TITLE
server: fix panic in api regions/meta

### PR DESCRIPTION
**Pull request by winkyao**
### What problem does this PR solve? 

```
func (t *tikvHandlerTool) getRegionsMeta(regionIDs []uint64) ([]RegionMeta, error) {
	regions := make([]RegionMeta, len(regionIDs))
	for i, regionID := range regionIDs {
		meta, leader, err := t.RegionCache.PDClient().GetRegionByID(context.TODO(), regionID)
		if err != nil {
			return nil, errors.Trace(err)
		}

		regions[i] = RegionMeta{
			ID:          regionID,
			Leader:      leader,
			Peers:       meta.Peers,
			RegionEpoch: meta.RegionEpoch,
		}

	}
	return regions, nil
}
```

if `t.RegionCache.PDClient().GetRegionByID(context.TODO(), regionID)` return nil, nil, nil. panic will throw in `meta.Peers`. when the tidb just start, the region cache can be empty, so the return value of GetRegionByID can be nil.

### Check List 

Tests 

 - Unit test

Side effects

 - Increased code complexity

### CODE REVIEW

File: server/http_handler.go [hunk 1]
```diff
@@ -518,6 +519,16 @@ func (t *tikvHandlerTool) getRegionsMeta(regionIDs []uint64) ([]RegionMeta, erro
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
+
+		failpoint.Inject("errGetRegionByIDEmpty", func(val failpoint.Value) {
+			if val.(bool) {
+				meta = nil
```
Comment (jackysp) [inline server/http_handler.go]: Why not `return` directly?

Comment (winkyao) [reply server/http_handler.go]: I wanna inject the case that meta is nil.

Comment (jackysp) [reply server/http_handler.go]: Is it better to inject it in `GetRegionByID`?

Comment (crazycs520) [reply server/http_handler.go]: @jackysp  `GetRegionByID` function is in `PD`, How to inject?

Comment (jackysp) [reply server/http_handler.go]: I think there is another implement in `mocktikv`.

**Comment by zimulala** [comment]
Please fix the CI.

**Pull request merged**